symbolic_pymc/tensorflow/meta.py

```diff
@@ -337,6 +337,13 @@ def process_node_def(cls, node_def, op_def, obj=None):
 
         return meta_node_def
 
+    @classmethod
+    def _metatize(cls, obj):
+        """Fix inputs from TF"""
+        obj.input = obj._reconstruct_sequence_inputs(obj.op_def,
```

Comment:
We should not have to alter TF objects for most&mdash;if not all&mdash;of the functionality in this project, so this assignment will require a lot of justification.

More importantly, this overloaded method isn't implemented correctly and will break `metatize` functionality for `tf.Operation` objects.  It needs to return a meta object.